pkg/constant/constant.go

```diff
@@ -1,13 +1,15 @@
 package constant
 
-var ErrorCode  = map[int]string{
+var ErrorCode = map[int]string{
 	0: "No Error",
 	1: "Client",
 	2: "Control Plane",
 	3: "Etcd Database",
 	4: "Executor",
 }
 
+const ConfigOctaviusHostMissingError = "Config Error!!!\nMandatory config CP_HOST is missing in Octavius Config file."
```

Comment:
I guess more error constants can be added for all the errors we are returning presently